evap/staff/templates/staff_semester_view.html

```diff
@@ -115,11 +115,15 @@ <h3 class="panel-title">{% trans "Actions" %}</h3>
                         <form id="form_{{ state }}" method="GET" action="{% url "staff:semester_course_operation" semester.id %}">
                             <table class="table table-striped vertically-aligned">
                                 <tbody>
-                                {% for course in courses %}
-                                    <tr>
-                                        {% include_staff_semester_view_course semester state course disable_if_archived %}
-                                    </tr>
-                                {% endfor %}
+                                {% if courses %}
+                                    {% for course in courses %}
+                                        <tr>
+                                            {% include_staff_semester_view_course semester state course disable_if_archived %}
+                                        </tr>
+                                    {% endfor %}
+                                {% else %}
+                                    <tr><td><i>{% trans "There are no courses in this state." %}</i><td></tr>
+                                {% endif %}
```

Comment:
you can use https://docs.djangoproject.com/es/1.9/ref/templates/builtins/#for-empty here